src/main/java/org/gridsuite/modification/server/entities/equipment/creation/BranchCreationEntity.java

```diff
@@ -64,21 +68,19 @@ public class BranchCreationEntity extends EquipmentCreationEntity {
     @Column(name = "connected2", columnDefinition = "boolean default true")
     private boolean connected2;
 
-    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
-    @JoinColumn(name = "current_limits_id1",
-        referencedColumnName = "id",
-        foreignKey = @ForeignKey(
-            name = "current_limits_id1_fk"
-        ), nullable = true)
-    private CurrentLimitsEntity currentLimits1;
-
-    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
-    @JoinColumn(name = "current_limits_id2",
-        referencedColumnName = "id",
-        foreignKey = @ForeignKey(
-            name = "current_limits_id2_fk"
-        ), nullable = true)
-    private CurrentLimitsEntity currentLimits2;
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
you removed the @JoinColumn ?

Reply:
Yes. Now that this is a list instead of a single object, a join column can't refeernce the whole list of temple entry.